docs/formats/tfvars-json.md

```diff
@@ -93,6 +93,7 @@ generates the following output:
       "string-1": "bar",
       "string-2": null,
       "string-3": "",
+      "string-special-chars": "\\.\u003c\u003e[]{}_-",
```

Comment:
This is a very valid point and after a little bit of digging I found that `orderedMap` doesn't allow for escaping which I sent them a PR (https://github.com/iancoleman/orderedmap/pull/15). We can proceed with this PR here and adjust accordingly when the other PR gets processed.